README.md

```diff
@@ -33,6 +33,13 @@ Data about food is of public interest and has to be open. The complete database
 
 * <https://world.openfoodfacts.org>
 
+## Roadmap
+* Extracting HTML templates from Perl code
```

Comment:
This should not be done by a newcomer, this is something we need to discuss and we need to agree on which templating system to use. We don't want someone to just come in and start to use one.